app.js

```diff
@@ -11,44 +11,75 @@ const catchphrasesEl = document.getElementById('catchphrases');
 const catchphraseInput = document.getElementById('catchphrase-input');
 const catchphraseButton = document.getElementById('catchphrase-button');
 
+
 // set state for how many times the user changes the head, middle, and bottom
+let headCount = 0;
+let middleCount = 0;
+let bottomCount = 0;
+
 // set state for all of the character's catchphrases
+let catchPhrases = [];
 
-headDropdown.addEventListener('change', () => {
+headDropdown.addEventListener('change', (e) => {
     // get the value of the head dropdown
+    const top = e.target.value;
+    //headEL.src = `./assets/${value}-head.png`;
+    headEl.style.backgroundImage = `url('../assets/${top}-head.png')`;
+    //headEL.src = `./assets/bird-head.png`;
+
 
     // increment the head change count state
+    headCount ++;
     
     // update the dom for the head (use style.backgroundImage on the bottomEl div instead of trying to set the .src -- it's NOT an img tag!)
 
     // update the stats to show the new count (refactor to/call displayStats() to do this work)
+    displayStats();
 });
 
 
-middleDropdown.addEventListener('change', () => {
-    // get the value of the middle dropdown
-
-    // increment the middle change count state
+middleDropdown.addEventListener('change', (e) => {
+        // get the value of the head dropdown
+    const mid = e.target.value;
+        //headEL.src = `./assets/${value}-head.png`;
+    middleEl.style.backgroundImage = `url('../assets/${mid}-middle.png')`;
+        //headEL.src = `./assets/bird-head.png`;
     
-    // update the dom for the middle (NOTE: use style.backgroundImage on the middleEl div instead of trying to set the .src -- it's NOT an img tag!)
-
-    // update the stats to show the new count (refactor to/call displayStats() to do this work)
+    
+        // increment the head change count state
+    middleCount ++;
+        
+        // update the dom for the head (use style.backgroundImage on the bottomEl div instead of trying to set the .src -- it's NOT an img tag!)
+    
+        // update the stats to show the new count (refactor to/call displayStats() to do this work)
+    displayStats();
 });
 
 
-bottomDropdown.addEventListener('change', () => {
-    // get the value of the bottom dropdown
-
-    // increment the bottom change count state
-    
-    // update the dom for the bottom (NOTE use style.backgroundImage on the bottomEl div instead of trying to set the .src -- it's NOT an img tag!)
-
-    // update the stats to show the new count (refactor to/call displayStats() to do this work)
+bottomDropdown.addEventListener('change', (e) => {
+       // get the value of the head dropdown
+    const btm = e.target.value;
```

Comment:
Be careful with using abbreviations here as it can sometimes make variable names/code harder to read. When coming up with variable names, remember that you're writing code for someone else to read. That someone might be you, weeks or months down the line, or someone entirely different. It should be clear what everything means, even if that means using longer variable/function names. #JuliTips 😂